packages/cli/src/__tests__/commands/__snapshots__/Version.test.ts.snap

```diff
@@ -20,7 +19,6 @@ Current platform        : TEST_PLATFORM
 Query Engine (Node-API) : libquery-engine ENGINE_VERSION (at sanitized_path/libquery_engine-TEST_PLATFORM.LIBRARY_TYPE.node, resolved by PRISMA_QUERY_ENGINE_LIBRARY)
 Migration Engine        : migration-engine-cli ENGINE_VERSION (at sanitized_path/migration-engine-TEST_PLATFORM, resolved by PRISMA_MIGRATION_ENGINE_BINARY)
 Introspection Engine    : introspection-core ENGINE_VERSION (at sanitized_path/introspection-engine-TEST_PLATFORM, resolved by PRISMA_INTROSPECTION_ENGINE_BINARY)
-Format Binary           : prisma-fmt ENGINE_VERSION (at sanitized_path/prisma-fmt-TEST_PLATFORM, resolved by PRISMA_FMT_BINARY)
```

Comment:
Random thought after reading the code here is that it's now impossible to specify an env var for the format binary (because removed)
So we also need to remove that from the docs after
- https://www.prisma.io/docs/concepts/components/prisma-engines#using-custom-engine-libraries-or-binaries
- https://www.prisma.io/docs/reference/api-reference/environment-variables-reference#prisma_fmt_binary

Reply:
That is actually a change in functionality that we probably need to look at in a bigger context - the same will be true if we move `getDmmf` to the wasm module in CLI :/

Reply:
Resolving this as we have started this discussion on [Slack](https://prisma-company.slack.com/archives/C02A1646TBR/p1671553918011699) already.

Reply:
We need to get to an agreed solution to that discussion though before we merge it.